app/views/employees/new.html.erb

```diff
@@ -21,7 +34,7 @@
 
     <div class="form-group">
         <%= label_tag 'private_number', 'Private Number:'%>
-        <%= form.text_field :private_number, class:'form-control', value: @private_number, disabled: true%>
+        <%= form.text_field :private_number, class:'form-control'%>
```

Comment:
I don't think manually setting the employee number is good. Why not automatically assign an employee number using a generator?

Reply:
We thought about that and we had a lot of trouble and because of time decided to implement it manually.